File: backend/internal/model/audio.go
package model

import (
	"time"
)

type Audio struct {
	ID         int64     `json:"id"`
	Title      string    `json:"title"`
	Artist     string    `json:"artist"`
	IPFSCID    string    `json:"ipfsCid"`
	OwnerAddr  string    `json:"ownerAddress"`
	Signature  string    `json:"signature"`
	UploadedAt time.Time `json:"uploadedAt"`
}

File: frontend/src/types/audio.ts
export interface Audio {
    id: number;
    title: string;
    cid: string;
    gradient?: string;
    link: string;
    owner_addr: string;
    signature?: string;
    uploaded_at?: string;
  }
  
  export interface Track {
    name: string;
    artist: string;
    link: string; 
    owner_addr: string;
  }